org.alloytools.alloy.application/src/main/java/templates/TestTemplateFile.vm

```diff
@@ -260,6 +293,19 @@ class Signature(metaclass = MetaSet):
         else: # Set
             return random_subset(self.atoms)
 
+    def possible_next_values(self):
+        ret = []
+        if self.multiplicity  == 0:  # Lone
```

Comment:
nit: can we use enum instead of "1" "2" "3"? This is kinda hard to read.

Reply:
Sure, I changed all multiplicities to Enum now.